packit/local_project.py

```diff
@@ -158,6 +164,28 @@ def refresh_the_arguments(self):
                 or self._parse_namespace_from_git_url()
             )
 
+    @contextmanager
+    def git_checkout_block(self, ref: str = None):
```

Comment:
What's the point of having ref=None, isn't that an error?

Reply:
It saves some ifs in the API...